mitosheet/mitosheet/api/get_ai_completion.py

```diff
@@ -12,9 +12,69 @@
 from mitosheet.ai.prompt import get_prompt
 
 
-import requests # type: ignore
+import requests
+
+from mitosheet.user.db import get_user_field, set_user_field
+from mitosheet.user.schemas import UJ_AI_MITO_API_NUM_USAGES, UJ_STATIC_USER_ID, UJ_USER_EMAIL
+from mitosheet.user.utils import is_pro # type: ignore
+
+OPEN_AI_URL = 'https://api.openai.com/v1/chat/completions'
+MITO_AI_URL = 'https://ogtzairktg.execute-api.us-east-1.amazonaws.com/Prod/completions/'
+
+def _get_ai_completion_data(prompt: str) -> Dict[str, Any]:
+        return {
+                "model": "gpt-3.5-turbo",
+                "messages": [{"role": "user", "content": prompt}],
+                "max_tokens": 7,
+                "temperature": .2,
+                'max_tokens': 200,
+                'stop': ['```']
+        }
+
+
+def _get_ai_completion_from_mito_server(user_input: str, prompt: str) -> str:
+        user_email = get_user_field(UJ_USER_EMAIL)
+        user_id = get_user_field(UJ_STATIC_USER_ID)
+        num_completions = get_user_field(UJ_AI_MITO_API_NUM_USAGES)
+        if num_completions is None:
+                num_completions = 0
+
+        pro = is_pro()
+
+        if not pro and num_completions >= 3:
```

Comment:
Update this to 20

Reply:
Maybe we can create a variable for the max number of Mito AI usages and then make sure it's in sync between the frontend and backend. Since we use it in a bunch of error messages. I'm sure we'll forget to update them all if we do